### ISSUE TITLE
App cannot reach the Netflix service

### ISSUE THREAD

**Issue by biosmanager** [longseespace/netflix_patch#1]
I tried using 4.16.3 on a Samsung Galaxy Tab S2 (T810) with a custom ROM Lineage OS 16.0 (systemless root with Magisk 19.3).
Playing any title results in this error message:

> "Sorry, we could not reach the Netflix service. Please try again later. If the problem persists, please visit the Netflix website (0013)."

The latest Netflix version from Google Play works fine, although no HD.

**Comment by JRdeLange** [comment]
I am having the same issue on my Xiaomi redmi note 8 pro. That first time I run it, it works fine, but if I try to view anything after closing the all I get code 0013. Have you found a solution yet?

**Comment by biosmanager** [comment]
No, I couldn't get any HD-patched version to work on my tablet. I'm currently using the official Netflix app, but the quality leaves something to be desired.

**Comment by rodricels** [comment]
I confirm, this patch is not working anymore. Netflix cut the network stream after a few seconds and thrown the 0013 error.

**Comment by fellipefgs** [comment]
Same problem here :(.
It seems that the error when you download the title is different, maybe is a little easier to solve?

**Comment by longseespace** [comment]
I cannot reproduce the issue with my device (Mi Projector 150"). Still working fine for me though.
Sorry cannot help you

**Comment by Destinyg133** [comment]
Can confirm it doesnt work anymore for me. Few seconds of runtime then 0013 error

**Comment by Valeria-h** [comment]
Did you try with the first version to have the new layout?

**Comment by Valeria-h** [comment]
Did you try the patch with the first version to come up with the new layout?

**Comment by Valeria-h** [comment]
Did you try with the first version that received the new layout?

**Comment by marcelloinfoweb** [comment]
Some Problem here!

**Comment by kugelvlitz** [comment]
Any progress? I'm getting the same Issue on a huawei phone. The few first seconds that it plays seems to be when the stream is sending just SD video, when it tries to ramp up the error is thrown. Same for downloads.
